subresource-loading.bs

```diff
@@ -211,8 +211,7 @@ bundle=] |web bundle| and [=fetch params=] |fetch params|:
    version of `1`, but this specification tracks what’s actually implemented in
    browsers, which still uses draft versions.
 
-1. Let |response| be the result of running <a spec="fetch">HTTP-network-or-cache
-   fetch</a> given |fetch params|.
+1. Let |response| be the result of running [=fetch=] given |request|.
```

Comment:
[Fetch](https://fetch.spec.whatwg.org/#concept-fetch) returns not a response but a [fetch controller](https://fetch.spec.whatwg.org/#fetch-controller). I guess we need to pass processResponse* algorithm(s) to fetch.
See https://fetch.spec.whatwg.org/#fetch-elsewhere

Reply:
Nice catch! Done.